model/project_parser.go

```diff
@@ -619,6 +625,7 @@ func evaluateBuildVariants(tse *taskSelectorEvaluator, tgse *tagSelectorEvaluato
 				}
 
 				var added []BuildVariantTaskUnit
+				// TODO: this might also be a mistake but unsure how to duplicate
```

Comment:
Remove. Or cover in a test case?

Reply:
ah something i did cover and forgot to remove, thank you